tools/tertiary-analysis/scanpy/scanpy-run-tsne.xml

```diff
@@ -68,21 +71,10 @@ PYTHONIOENCODING=utf-8 scanpy-run-tsne.py
       </when>
     </conditional>
 
-    <conditional name="do_plotting">
-      <param name="plot" type="boolean" checked="false" label="Make tSNE plot"/>
-      <when value="true">
-        <expand macro="output_plot_params"/>
-        <param name="color_by" argument="--color-by" type="text" value="louvain" label="Color by attributes, comma separated strings"/>
-      </when>
-      <when value="false"/>
-    </conditional>
   </inputs>
 
   <outputs>
     <data name="output_h5" format="h5" from_work_dir="output.h5" label="${tool.name} on ${on_string}: tSNE object"/>
-    <data name="output_png" format="png" from_work_dir="output.png" label="${tool.name} on ${on_string}: tSNE plot">
```

Comment:
Why are we removing the plot from a method whose main purpose is producing a plot (for most people I mean)?